core/src/main/kotlin/org/sert2521/sertain/motors/MotorController.kt

```diff
@@ -165,47 +153,59 @@ class MotorController<T : MotorId>(
                 field = value
             }
 
-    fun setPercentOutput(output: Double) {
+    val percentOutput: Double
+        get() = ctreMotorController.motorOutputPercent
+
+    var position: Int
+        get() = ctreMotorController.getSelectedSensorPosition(0)
+        set(value) {
+            ctreMotorController.selectedSensorPosition = value
+        }
+
+    fun position(unit: MetricUnit<Angular>) =
+            MetricValue(encoder!!.ticks, position.toDouble()).convertTo(unit)
+
+    val velocity: Int
+        get() = ctreMotorController.getSelectedSensorVelocity(0)
+
+    fun velocity(unit: CompositeUnit<Per, Angular, Chronic>) =
+            MetricValue(encoder!!.ticksPerSecond, velocity.toDouble()).convertTo(unit)
+
+    fun targetPercentOutput(output: Double) {
```

Comment:
```suggestion
    fun setPercentOutput(output: Double) {
```
This should not have a "target" in the name because there is no closed-loop control here. The change is immediate.

Reply:
All set